indico/modules/rb/models/locations.py

```diff
@@ -162,18 +162,22 @@ def get_equipment_by_name(self, name):
         return self.equipment_types.filter_by(name=name).first()
 
     def get_buildings(self):
-        building_rooms = defaultdict(list)
+        from indico.modules.rb_new.schemas import rooms_schema
```

Comment:
I would add a new `get_buildings()` util function (or even method in the RH used for it) in the new RB module and keep this one as it is (and eventually get rid of it). It's quite specific to the room list and doesn't really need to be in the model.

Reply:
Also, I don't think it makes sense to have this limited to a specific location. As long as a room is active it should be used IMO.

Speaking of that: Are we already making sure to not include rooms that are disabled in the room list etc?

Reply:
We check against `is_active` property of a room